lib/tictactoe/player.ex

```diff
@@ -0,0 +1,12 @@
+defmodule TicTacToe.Player do
+  alias TicTacToe.ConsoleIO
+
+  defstruct type: nil,
+            name: nil,
+            marker: nil,
+            turn_msg: nil
+
+  def select_cell, do: ConsoleIO.input()
+
+  defimpl(String.Chars, do: def(to_string(player), do: player.name))
```

Comment:
lol I think that was something that Credo told me to use in an earlier implementation because I was trying to print out the player's name by calling `player` instead of `player.name`. Removing this, thanks for the heads up!